templates/Collector/db.html.twig

```diff
@@ -90,6 +90,13 @@
             connectionName: request.query.get('connection'),
             query: request.query.get('query')
         })) }}
+        {% elseif 'run' == page %}
```

Comment:
I don't like that most of the lines are duplicated now. Something like this could be done instead:
```php
{% if page in ['explain', 'page'] %}
  {{ render(controller('Doctrine\\Bundle\\DoctrineBundle\\Controller\\ProfilerController::' ~ page ~ 'Action', {
            token: token,
            panel: 'db',
            connectionName: request.query.get('connection'),
            query: request.query.get('query')
        })) }}
```